lib/enumerize/activerecord.rb

```diff
@@ -52,11 +52,17 @@ def write_attribute(attr_name, value, *options)
       # Support multiple enumerized attributes
       def becomes(klass)
         became = super
+        klass = self.class unless klass.respond_to?(:enumerized_attributes)
         klass.enumerized_attributes.each do |attr|
-          # Rescue when column associated to the enum does not exist.
+          # Rescue when column associated to the enum does not exist,
+          # or when destination model does not include Enumerize.
           begin
-            became.send("#{attr.name}=", send(attr.name))
+            if became.respond_to?(setter = "#{attr.name}=")
+              became.send(setter, send(attr.name))
+            end
           rescue ActiveModel::MissingAttributeError
+          rescue ActiveRecord::SerializationTypeMismatch
+            became.send(setter, send(attr.name).to_ary)
```

Comment:
is there any reason why you added `to_ary` and rescue from `ActiveRecord::SerializationTypeMismatch` here?

Reply:
Without this rescue and `.to_ary` then when running:
```
bundle exec rake TEST=lib/enumerize/activerecord.rb
```
then there is this failure:
```
ActiveRecord::SerializationTypeMismatch: can't serialize `interests`: was supposed to be a Array, but was a Enumerize::Set.
```

Reply:
Right, I see, thanks for explaining! I'm not sure how I feel about setting something when we already got `ActiveRecord::SerializationTypeMismatch` error. If we got this it means that `klass` doesn't have proper attribute (with proper type). What do you think if we just ignore such attributes as well?